check_docker.py

```diff
@@ -288,6 +289,7 @@ def print_results():
 
     #############################################################################################
     args = process_args(argv[1:])
+    checks = 0
```

Comment:
Testing for 'checks' is a great idea. We should really add a unit test that, I can take care of that if you don't want to. I am happy top merge this function anytime.